### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by maxcanna**
Ref. #3

### CODE REVIEW

File: redis.js [hunk 1]
```diff
@@ -13,12 +14,13 @@ if (process.env['REDISCLOUD_URL']) {
     redisClient.on('error', console.error);
     redisClient.on('ready', function () {
         console.info('Connected to redis');
+        module.exports.emit('ready');
```
Comment (maxcanna) [inline redis.js]: `redis` is already defined (it's the redis module) but I got the point, thanks

File: redis.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const redis = require('redis');
 const async = require('async');
+const events = require('events');
```
Comment (maxcanna) [inline redis.js]: Thanks for finding that

**Comment by maxcanna** [comment]
@demerzel3 I followed your suggestions. I rebased but kept the fixups.

Here are some screens to ease the review:

|Password confirmation and mismatch check|
|-|
|schermata 2017-02-05 alle 18 42 50|

|Enabled branch and syntax check|
|-|
|schermata 2017-02-05 alle 18 44 34|

File: views/admin.ejs [hunk 3]
```diff
@@ -29,7 +29,8 @@
                             %>
                             <form method="post" action="/admin">
                                 <input name="username" type="text" class="form-control" placeholder="Username" required value="<%= adminCredentials.username %>"><br>
-                                <input name="password" type="password" class="form-control" placeholder="Password" required value="<%= adminCredentials.password %>"><br>
+                                <input id="password" name="password" type="password" class="form-control" placeholder="Password" required ><br>
+                                <input id="password-confirm"type="password" class="form-control" placeholder="Password confirm" required ><br>
```
Comment (demerzel3) [inline views/admin.ejs]: missing space before `type`

File: views/admin.ejs [hunk 4]
```diff
@@ -91,5 +92,14 @@
                 location.reload();
             });
         });
+
+        document.getElementById('password-confirm').onchange = function () {
```
Comment (demerzel3) [inline views/admin.ejs]: This handler should probably be fired on change of both password fields, imagine this scenario:
- I type the password
- I type the password confirm
- the system tells me they are different
- I type the confirm again, I am extra sure I got it right, still have the error
- I type password again, it matches now but there's still an error

Comment (demerzel3) [inline views/admin.ejs]: Also using `addEventListener` is better than setting the handler directly.

Comment (maxcanna) [inline views/admin.ejs]: @demerzel3 always one step ahead! Thanks for the suggestions 😀

**Pull request merged**